CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Changelog for Elixir v1.9
 
+## Releases
+
+The main feature in Elixir v1.9 is the addition of releases. A release is a self-contained directory with your application code, all of its dependencies, plus the whole Erlang VM and runtime. Once a release is assembled, it can be packaged and deployed to a target, as long as the target runs on the same operating system (OS) distribution and version as the machine running the `mix release` command.
+
+You can start a new project and assemble a release for it in 3 easy steps:
+
+    $ mix new my_app
+    $ cd my_app
+    $ MIX_ENV=prod mix release
+
+A release will be assembled in `_build/prod/rel/my_app`. Inside the release, there will be a `bin/my_app` file which is the front-end to your system. It supports multiple commands, such as:
```

Comment:
The "front-end" term has a lot of connotations these days, maybe "the main interface to your system" or similar?